src/components/expanded-state/unique-token/ZoomableWrapper.js

```diff
@@ -69,25 +76,39 @@ const MIN_IMAGE_SCALE = 1;
 const THRESHOLD = 250;
 
 export const ZoomableWrapper = ({
-  animationProgress: givenAnimationProgress,
+  animationProgress: givenAnimationProgress = undefined,
   children,
+  hasShadow = true,
   horizontalPadding,
   aspectRatio,
   borderRadius,
   disableAnimations,
-  onZoomIn,
-  onZoomOut,
+  disableEnteringWithPinch,
+  hideStatusBar = true,
+  onZoomIn = () => {},
+  onZoomInWorklet = () => {},
+  onZoomOut = () => {},
+  onZoomOutWorklet = () => {},
+  opacity,
+  yOffset = 85,
+  xOffset: givenXOffset = 0,
   yDisplacement: givenYDisplacement,
+  width,
+  height,
 }) => {
   // eslint-disable-next-line react-hooks/rules-of-hooks
   const animationProgress = givenAnimationProgress || useSharedValue(0);
   // eslint-disable-next-line react-hooks/rules-of-hooks
   const yDisplacement = givenYDisplacement || useSharedValue(0);
 
-  const { height: deviceHeight, width: deviceWidth } = useDimensions();
+  let { height: deviceHeight, width: deviceWidth } = useDimensions();
 
-  const maxImageWidth = deviceWidth - horizontalPadding * 2;
-  const maxImageHeight = deviceHeight / 2;
+  if (!hideStatusBar) {
+    deviceHeight = deviceHeight - safeAreaInsetValues.top;
```

Comment:
I don't think it's ok. Can you introduce a variable e.g., `deviceHeightWithMaybeHiddenStatusBar`? Probably name cam be better

Reply:
sounds good!